INSTALL-STANDALONE.md

```diff
@@ -24,59 +20,36 @@ The installation steps below will allow you to set-up the application as a stand
 5. Tap _"Always allow from this computer"_.
 6. Tap **OK**.
 
-## 2. Building the onecourse app
+## 2. Building the Pehla Class app
 
 1. Ensure you have **Android Studio** installed. To do this, you can [follow the steps in this helpful guide](https://developer.android.com/studio/install.html).
+     
+2. Start Android Studio and import the project into it.
 
-2. Configure the _ROOT_FOLDER_ environment variable and _ANDROID_HOME_ environment variable based on the location of the Android SDK:  
-   
-   Follow these steps for macOS (Mavericks or later) or Linux-based system:
-	
-		export ROOT_FOLDER=onecourse
-		
-		export ANDROID_HOME=[PATH/TO/ANDROID_SDK]
-		
-   Follow these steps for a Windows 7 or later system:
+3. To change the app locale, just select the desired language from the build variant.
```

Comment:
In order to build the app for a specific language, select the desired language from the build variant.